core/vil1/examples/vil1_print_section.cxx

```diff
@@ -32,9 +32,7 @@ to_hex(std::ostream & os, unsigned char c)
 }
 
 // usage : vil1_print_section image x0 y0 w h
-int
-main(int argc, char ** argv)
-{
+int main(int /*argc*/, char **argv) {
   assert(argc == 6);
```

Comment:
This whole commit should be significantly revised by running clang-tidy in debug mode because most parameters are checked in assertions.